fix(pages): removes deprecated use of input/write_access view

mrclay opened the pull request:
Deprecation notices will only occur if a dev has extended forms/pages/edit or input/write_access

Fixes #8327

hypeJunction commented:
LGTM

juho-jaakkola commented on mod/pages/views/default/forms/pages/edit.php:
```diff
@@ -44,6 +44,10 @@
 		);
 		if ($input_view === 'input/access' || $input_view === 'input/write_access') {
+
+			// we'll instead be using use the "purpose" argument of input/access
```
The comment is a bit weird

mrclay committed:
fix(pages): removes deprecated notices regarding input/write_access

Deprecation notices will only occur if a dev has extended forms/pages/edit
or input/write_access

Fixes #8327

mrclay commented:
This was reworked to not break BC with plugins that extend input/write_access.
